mapper/census.go

```diff
@@ -216,7 +216,7 @@ func CreateCensusDatasetLandingPage(ctx context.Context, req *http.Request, base
 		if latestVersionNumber != version.Version && hasOtherVersions {
 			p.DatasetLandingPage.Panels = append(p.DatasetLandingPage.Panels, datasetLandingPageCensus.Panel{
 				DisplayIcon: true,
-				Body:        helper.Localise("HasNewVersion", lang, 1),
+				Body:        helper.Localise("HasNewVersion", lang, 1, strconv.Itoa(latestVersionNumber)),
```

Comment:
Think you need to use `latestVersionURL`, that param will just be a number (converted to a string)

Reply:
fair - should have read the localiser more thoroughly